src/main/java/hudson/plugins/accurev/AccurevSCM.java

```diff
@@ -370,9 +372,16 @@ public void checkout(@Nonnull Run<?, ?> build, @Nonnull Launcher launcher, @Nonn
                          @Nonnull TaskListener listener, @CheckForNull File changelogFile,
                          @CheckForNull SCMRevisionState baseline) throws IOException, InterruptedException {
 //        TODO: Implement SCMRevisionState?
+    	final AccurevServer server = getServer();
+        boolean serverDisabled = server != null && server.isServerDisabled();
+        if (!serverDisabled) {
```

Comment:
move this check into the the Abstract Delegate there you already have access to the server variable

Reply:
Changes made and pushed.